as/cNFT/assembly/market.ts

```diff
@@ -34,3 +35,20 @@ export function get_bids(tokenId: string): Map<string, Bid> {
 export function get_bidder_bids(accountId: string): Bid[] {
     return persistent_market.get_by_bidder(accountId)
 }
+
+
+@nearBindgen
+export function remove_bid(tokenId:string, accountId: string): void {
+    persistent_market.remove(tokenId, accountId)
+}
+
+@nearBindgen
+export function set_bid_shares(
+    token_id: string,
+    prev_owner: u16,
+    creator: u16,
+    owner: u16
+  ): void {
+    const new_bid_shares = new BidShares(prev_owner, creator, owner);
```

Comment:
Same as other comment; We want `owner`, `creator` and `prev_owner` to be directly integrated into Royalty+Payout functionalities that are already implemented here. Actually I think the only use case that need to be added is `prev_owner` cause the other two are already part of it.